configs/edison/config.20180129.DECKv1b_piControl.ne30_oEC.edison

```diff
@@ -32,13 +43,6 @@ oceanHistorySubdirectory = archive/ocn/hist
 # subdirectory for sea ice history files
 seaIceHistorySubdirectory = archive/ice/hist
 
-# names of namelist and streams files, either a path relative to baseDirectory
-# or an absolute path.
-oceanNamelistFileName = run/mpas-o_in
-oceanStreamsFileName = run/streams.ocean
-seaIceNamelistFileName = run/mpas-cice_in
-seaIceStreamsFileName = run/streams.cice
-
```

Comment:
@xylar: why are these removed? Since the `baseDirectory` does not have 'run/' in it, I think we need to add it to the namelist and streams filenames.

Reply:
oh never mind, they are right below..